<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationIdDialog</class>
 <widget class="QDialog" name="StationIdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station ID</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Identification</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>450</width>
       <height>135</height>
      </rect>
     </property>
     <property name="title">
      <string>Legal</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User call sign</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Message ID prefix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(3 characters max)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cUserCallSign">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>25</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="cUserName">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Transmit PST</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cUserMessageIdPrefix">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>75</y>
        <width>55</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>100</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="cNewUser">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cDeleteUser">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>45</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="cCurrentProfile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>20</y>
       <width>295</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Profile</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>195</y>
       <width>446</width>
       <height>146</height>
      </rect>
     </property>
     <property name="title">
      <string>Tactical</string>
     </property>
     <widget class="QLineEdit" name="cTacticalName">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cNewTactical">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>25</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cTacticalCallSign">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>55</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>105</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(3 characters max)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cTacticalMessageIdPrefix">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>105</y>
        <width>55</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Message ID prefix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tactical name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>135</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tactical call sign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cDeleteTactical">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>50</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cUseTactical">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use tactical call for all BBS interaction</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>BBS Logins</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Signatures</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="cOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>405</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cCancel">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>405</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cShowAtStartup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>225</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show this form on startup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cOK</sender>
   <signal>clicked()</signal>
   <receiver>StationIdDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cCancel</sender>
   <signal>clicked()</signal>
   <receiver>StationIdDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
